src/Tasks/ContentReviewEmails.php

```diff
@@ -93,6 +106,17 @@ protected function notifyOwner($ownerID, SS_List $pages)
         // Prepare variables
         $siteConfig = SiteConfig::current_site_config();
         $owner = Member::get()->byID($ownerID);
+
+        if (!$this->isValidEmail($owner->Email)
+            || !$this->isValidEmail($siteConfig->ReviewFrom)) {
+            $invalidEmail = !$this->isValidEmail($owner->Email)
+                            ? $owner->Email
+                            : $siteConfig->ReviewFrom;
+            $this->invalid_emails[] = (string) $invalidEmail;
+
+            return;
+        }
```

Comment:
```suggestion
        if (!$this->isValidEmail($owner->Email)) {
            $this->invalid_emails[] = $owner->Email;
            return;
        }
```
We don't need to check the `$siteConfig->ReviewFrom` value for every member, that's going to be the same regardless. Just check that once before we even start looking at members.

Reply:
Also, for any email addresses that are just empty, this won't provide useful output... is there a way we can gracefully handle that? Maybe something like `$this->invalid_emails[] = $owner->Name . ': ' . $email`?

Reply:
Yeah, I also thought about to add more details about owner, but thought, probably it will be too bulky. And another problem, completely "Empty user" without name, surname and email.
DONE.